city_scrapers/spiders/chi_ssa_60.py

```diff
@@ -0,0 +1,129 @@
+        for i in range(-self.months_backward, self.months_forward + 1):
+            dt = datetime.strftime(datetime.now().date() + relativedelta(months=i), "%Y-%m")
+            requests.append(
+                Request(
+                    "https://northrivercommission.org/events/{}/".format(dt), callback=self.parse
+                )
+            )
+        return requests
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        event_list = list()
+        for item in response.xpath("//script[@type='application/ld+json']/text()"):
+            event_list = json.loads(self._clean(item.get()))
+
+        for item in event_list:
+            title = self._parse_title(item)
+            if title is None:
+                continue
+
+            meeting = Meeting(
+                title=title,
+                description=self._parse_description(item),
+                classification=self._parse_classification(title),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=False,  # Found no events of interest scheduled for all day
+                time_notes="",
+                location=self._parse_location(item),
+                links=list(),
+                source=item["url"],
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        allowed = ['meeting', 'committee', 'advisory council']
+        if not any([a in item['name'].lower() for a in allowed]):  # Not a meeting type we want
+            return None
+        return item['name']
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        desc = html.unescape(item["description"])
+        removeables = [
+            "<p>",
+            "</p>",
+            "\\n",
+            "\\",
+        ]
+        for r in removeables:
+            desc = desc.replace(r, "")
+        return desc
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        title = title.lower()
+        if 'advisory' in title:
+            return ADVISORY_COMMITTEE
+        elif 'committee' in title:
+            return COMMITTEE
+        # elif 'economic' in title or 'environmental' in title:
+        elif any(s in title for s in ['economic', 'environmental', 'membership', 'commission']):
+            return COMMISSION
+        else:
+            return NOT_CLASSIFIED
+
+    def _parse_time(self, item, index):
+        # index must be either 'startDate' or 'endDate'
+        return datetime.strptime(item[index][:-6], "%Y-%m-%dT%H:%M:%S")
+
+    def _parse_start(self, item):
```

Comment:
If `_parse_start` and `_parse_end` are essentially the same thing, it might make sense to just call `_parse_time` twice in the `parse` method. The methods are just a starting point, so if some of them are redundant it's fine to consolidate them

Reply:
Sounds good! I wasn't sure if I should prioritize uniformity between spiders or just sensible code in general.